wpilibj/src/main/java/edu/wpi/first/wpilibj/vision/VisionPipeline.java

```diff
@@ -0,0 +1,29 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2016-2018 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package edu.wpi.first.wpilibj.vision;
+
+import org.opencv.core.Mat;
+
+/**
+ * A vision pipeline is responsible for running a group of
+ * OpenCV algorithms to extract data from an image.
+ *
+ * @see VisionRunner
+ * @see VisionThread
+ *
+ * @deprecated Replaced with edu.wpi.first.cameraserver.vision.VisionPipeline
```

Comment:
should we be doing edu.wpi.first.cameraserver.vision or just edu.wpi.first.vision?

Reply:
Hmm. I don't really have a preference either way. .vision is a little shorter which would let me not require the line length change.

Reply:
Actually, looking again, the stuff in .vision is special for opencv processing, and the package-info would be completely different. So I like having CameraServer and that stuff in separate packages.